lib/lightstep/span_context.rb

```diff
@@ -1,12 +1,31 @@
+#frozen_string_literal: true
+
 module LightStep
   # SpanContext holds the data for a span that gets inherited to child spans
   class SpanContext
-    attr_reader :id, :trace_id, :baggage
+    attr_reader :id, :trace_id, :trace_id16, :sampled, :baggage
```

Comment:
The SpanContext maintains an 8 byte and 16 byte trace id. The 16 byte trace id is either the original, or an 8 byte id padded with zeroes on the left.